com.unity.render-pipelines.high-definition/Runtime/Lighting/Reflection/HDProbe.cs

```diff
@@ -633,15 +633,11 @@ private Vector3 ComputeCapturePositionWS()
 
         private void QueueSHBaking()
         {
-            var hdrp = (HDRenderPipeline)RenderPipelineManager.currentPipeline;
-            if (hdrp != null)
-            {
-                if (!hdrp.currentPlatformRenderPipelineSettings.supportProbeVolume) return;
-            }
-            else
-            {
+            if (!(RenderPipelineManager.currentPipeline is HDRenderPipeline hdrp))
```

Comment:
Maybe we can simplify with `HDRenderPipeline.asset != null` and/or `HDRenderPipeline.currentPlatformRenderPipelineSettings != null` ?